### ISSUE TITLE
Hypernode keeps asking for user app's password on SSH 

### ISSUE THREAD

**Issue by joostvanveen** [ByteInternet/hypernode-vagrant#198]
Something really weird.

Set up a local Hypernode yesterday. All worked well. Added this to ~/.ssh/config as per spec:
```
Host someserver.hypernode.local
Hostname someserver.hypernode.local
User app
StrictHostKeyChecking no  
```
I could simply SSH into Hypernode using `ssh someserver.hypernode.local`.

Logged out of Hypernode, left Hypernode running overnight. Now, when trying to SSH in Hypernode asks me for `app@someserver.hypernode.local's password`. I can SSH in and then do `sudo su - app`, but I would also like to use Sequel Pro via SSH.

Tried `vagrant halt && vagrant up`, but no luck.

Anything I am missing?

**Comment by joostvanveen** [comment]
Just found out wat happened, problem solved. Now I can login again using just 
```
ssh someserver.hypernode.local
```

**Keys were stored in Hypernode, but not used on login**
When I tried to connect using `ssh someserver.hypernode.local -vvv` I saw that SSH was trying my SSH keys, but not all of them.

So I looked if any of my local keys were registered in my Hypernode box.
```
vagrant ssh
sudo su - app
cat ~/.ssh/authorized_keys
```

I saw two of my keys were there, probably stored when I first logged in.

**Added IdentityFile to SSH config**
I edited my local ~/.ssh/config and added the identityFile to use in the future. Problem solved.
```
Host someserver.hypernode.local
Hostname someserver.hypernode.local
User app
IdentityFile ~/.ssh/my_special_key
StrictHostKeyChecking no 
```
